packages/core/src/secret/index.ts

```
@@ -283,94 +283,97 @@ async function batchGetKeys(
-  prefix.split('/').forEach((pathComponent) => {
-    if (pathComponent === 'm') {
-      return;
+  // Generate master key
+  let key: IBip32ExtendedKey = await deriver.generateMasterKeyFromSeed(
+    seedBuffer,
+  );
+
+  // Process prefix path components
+  const prefixComponents = prefix.split('/').filter((p) => p !== 'm');
+  const indices = prefixComponents.map((p) =>
+    p.endsWith("'") ? parseInt(p.slice(0, -1), 10) + 2 ** 31 : parseInt(p, 10),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)